dvc/repo/experiments/utils.py

```diff
@@ -191,3 +191,16 @@ def check_ref_format(scm: "Git", ref: ExpRefInfo):
             f"Invalid exp name {ref.name}, the exp name must follow rules in "
             "https://git-scm.com/docs/git-check-ref-format"
         )
+
+
+def fetch_all_exps(scm: "Git", url: str, progress: Optional[Callable] = None):
```

Comment:
Excuse me, do we need to download all exps instead of only the experiment listed after `--rev`?

Reply:
Yes, this is intended to make clone behave the same for exp refs as it behaves for `--no-single-branch` - it will fetch all available branches and tags (so we want to fetch all available experiment refs as well).